pkg/util/mount/fake.go

```diff
@@ -29,6 +29,7 @@ import (
 type FakeMounter struct {
 	MountPoints []MountPoint
 	Log         []FakeAction
+	Filesystem  map[string]FileType
```

Comment:
This type should be moved to a testing/ subdir so it doesn't get linked into non-test code.  Won't block this PR on it.

Reply:
OK. I'll change a lot of go imports and BUILD files, I'll create another PR for this.